docs/wiki/_Sidebar.md

```diff
@@ -0,0 +1,70 @@
+
+- [Home](Home)
+  - [What is Bison Wallet](Home#what-is-bison-wallet)
+  - [What is DCRDEX?](Home#what-is-dcrdex)
+  - [Tatanka Mesh](Home#tatanka-mesh)
+  - [Project History](Home#project-history)
+- [Getting Started](Getting-Started)
+  - [Download and Install](Download-and-Install)
+  - [Quick Configuration](Quick-Configuration)
+  - [Creating a DCRDEX Sever Account](Creating-a-DCRDEX-Server-Account)
+    - [Fidelity Bonds](Creating-a-DCRDEX-Server-Account#fidelity-bonds)
+- [Using Bison Wallet](Using-BIson-Wallet)
```

Comment:
BIson -> Bison

Reply:
the main page for Using Bison Wallet was misspelled, has been fixed and al links updated on the next push